---
title: "Why are males and females different, and why don't we know?"
collection: research
permalink: /research/quantitative-genetics
excerpt: 'Selection can drive genetic differentiation between the two sexes of a single species in coding and non-coding regions, and many questions remain regarding the relative importance and types of selection that are driving this differentiation, how genetic variation is maintained, and how the genetic architecture of traits impacts genetic divergence. The lab advocates for and employs the use of an integrative and holistic framework that considers when genes are able to respond to sex-specific selection, which tissues are likely to expereince different forms of sex-specific selection, when during the life cycle different forms of sex-specific selection are likely to create conflict, and whether proxies of sex-specific selection reflect the true strength of sexual selection.'
---

Sexual dimorphism (differences in traits between males and females) is one of the most prevalent, and often the most extreme, example of phenotypic variation within species, and arises primarily from genomic variation that is shared between females and males. Many sexual dimorphisms arise through sex differences in gene expression, and sex-biased expression is one way that a single, shared genome can generate multiple, distinct phenotypes. Although many sexual dimorphisms are expected to result from sexual selection, and there has been considerable research done looking into the possible role of sexual selection to explain sex-specific traits, the role of sexual selection in the evolution of sexually dimorphic gene expression remains difficult to differentiate from other forms of sex-specific selection.

In an attempt to address this problem, the lab is pursuing a holistic and integrative framework for studying sex-specific selection and transcriptome evolution. Here we combine an understanding of the behavioral ecology of sex-specific selection with the developmental and genomic biology of the organism. To begin fully deciphering any relationship present between sex-specific selection and sexually dimorphic transcriptomes we consider when genes are able to respond to sex-specific selection pressures, which tissues are likely to experience different forms of sex-specific selection, when during the life cycle different forms of sex-specific selection are likely to create conflict, and whether proxies of sex-specific selection reflect true strengths of sexual selection.

To accomplish this the lab uses tools such as RNA sequencing, parentage analysis, mesocosms, and behavioural coding with pipefish as a model system. Pipefish are a prominent example of a sex-role-reversed system where females compete for access to males who then care for the eggs in a brooding structure. They provide a particularly useful model system for this project as the different species are representative of both the spectra of mating systems and the spectra of sexual dimorphisms with females of some species possessing extremely elaborate ornamentation and nearly identical males and females in others. Additionally, pipefish posses no distinct sex chromosomes, meaning there are no parts of the genome inherited by only one sex and little is known about their sexual development. This makes sex-biased gene expression an especially likely way to resolve conflict. Currently the lab is working with <i>Syngnathus</i> spp. (pictured below) located throughout the States and <i>Stigmatopora</i> spp. found across New Zealand and Australia. 

<img align="center" src='/images/syngnathussp.jpg' width="110%" height="110%"> 

### Some relevant papers include:

Tosto NM, Beasley ER, Wong BBM, Mank JE, Flanagan SP. 2023. The roles of sexual selection and sexual conflict in shaping patterns of genome and transcriptome variation. Nature Ecology & Evolution. https://doi.org/10.1038/s41559-023-02019-7. <a href="https://rdcu.be/c8jST"  target="_blank" rel="noopener" style="color: rgb(82, 173, 200)">link to paper</a>

Anderson AP, Flanagan SP, Rose E, Jones AG. 2020. The estrogen-responsive transcriptome of female secondary sexual traits in the Gulf pipefish, Journal of Heredity, esaa008

Lerch B, Dautel KA, Brewer S, Liang A, Siewe N, Flanagan SP. 2020. Space, density, and extra-pair matings have opposing impacts on male and female reproductive success. Population Ecology.

Flanagan SP, and Jones AG. 2017. Constraints on the <em>F</em><sub>ST</sub>-heterozygosity outlier approach. Journal of Heredity. esx048. doi: 10.1093/jhered/esx048  <a title="Flanagan and Jones 2017 JHeredity" href="https://sarahpflanagan.files.wordpress.com/2018/01/flanagan-and-jones-2017-jheredity.pdf" target="_blank" rel="noopener">pdf</a>, <a href="https://academic.oup.com//jhered/article/doi/10.1093/jhered/esx048/3806805/Constraints-on-the-FSTHeterozygosity-Outlier?guestAccessKey=bb60bd3f-bdc2-4673-9b74-92efefd04db0" target="_blank" rel="noopener">link to paper</a>, <a href="https://sarahpflanagan.wordpress.com/2017/06/07/finding-limitations-with-common-analysis-methods-my-new-paper/" target="_blank" rel="noopener">my summary</a>, <a href="https://github.com/spflanagan/fsthet_analysis" target="_blank" rel="noopener">associated code</a>, <a href="http://datadryad.org/handle/10255/dryad.103085" target="_blank" rel="noopener">data</a>

Flanagan SP, and Jones AG. 2017. Genome-wide selection components analysis in a fish with male pregnancy. Evolution.  71: 1096â€“1105. DOI: 10.1111/evo.13173
