---
title: "How is population decline influenced by evolutionary mechanisms?"
excerpt: "The Flanagan Lab uses population genetics tools to improve our understanding of the genetic dynamics and factors that contribute to population persistence and fitness. Our lab is using a variety of molecular tools to investigate the evolutionary mechanisms impacting New Zealand populations of kororā, pipefish, and vegetation impacted by dieback disease."
permalink: /research/cons-gen
show_date: false
collection: research
---

We are interested in studying what causes populations to decline. Variation between populations is driven by a number of evolutionary mechanisms, for example, the availability of prey species, introduction of disease, inbreeding depression and intraspecific phenotypic variation. Evaluating the ecological and evolutionary interactions within populations is important for informing effective conservation management of endemic species.

To conserve endemic species in Aotearoa, we can use population genetics tools to characterise population structure, estimate population sizes, and identify local adaptation. When informing conservation management efforts, understanding evolutionary mechanisms is important to get the full picture of factors impacting population persistence. The Flanagan lab is exploring non-invasive molecular methods for assessing Pōhatu’s kororā population's genetic variation. We are particularly interested in breeding pairs where parents are a mix of the white-flippered and little blue morphotypes. We hope to use molecular methods to better understand the persistence of the white-flippered morphotype, which is unique to the Canterbury region, and explore the impacts phylogenetic constraints may be having on the colony. Additionally, we are working on modelling the distribution of the wide-bodied pipefish, <i>Stigmatopora nigra</i>, and using genetic analysis to compare population structure across its range. By comparing phenotypic and genotypic data of <i>S. nigra</i> across Aotearoa we will evaluate different selection pressures, assess the evolutionary mechanisms that drive local populations, and expand on the currently limited understanding of <i>S. nigra</i>.

Understanding factors of population decline is critical for conserving the species they affect. The Flanagan lab is currently focused on diet and disease as agents of decline. In collaboration with Mana Whenua, Pohatu Penguins and the LaRue and ConSERT labs we are utilizing long-read nanopore sequencing to assess the diet of the Pohatu population of kororā as a potential driver of local population decline. Using non-invasive fecal and molt feather samples to develop a baseline understanding of local diet, this project aims to facilitate community engagement in genetic sampling of local populations. Collaboration with the Banks Peninsula community will be further supported through the development of remote sequencing using the portable MinION device (Oxford nanopore technologies). In addition to diet in kororā populations, the Flanagan lab is interested in the coevolution of invasive diseases, specifically <i>Phytophthora</i>, and the plants they impact. In collaboration with Claudia Meisrimler, we are looking at <i>Phytophthora cinnamomi</i>, which causes dieback non-discriminantly across endemic and economically important flora and is a close relative to <i>P. agathidicida</i>, the source of Kauri dieback. Improving our understanding of the <i>P. cinnamoni</i> genome and how it co-evolves with crop species will allow for better informed management and conservation programmes aimed at tackling dieback diseases.

<img src="/images/E_minor_albosignata_chick.JPG">
White flippered kororā chick in a nest box at Pohatu. Credit Lucy Howell


<img src="/images/P.Pluvialis.png">
*Phytophthora Pluvialis*. Credit Sophie Eccersall

<br>

### Some relevant papers include:

<span style="color:#239B56">Howell LA</span>, LaRue M, <strong>Flanagan SP</strong> (2021) Environmental DNA as a tool for monitoring Antarctic vertebrates. New Zealand Journal of Zoology. DOI: 10.1080/03014223.2021.1900299. <a href="https://doi.org/10.1080/03014223.2021.1900299" target="_blank" rel="noopener" style="color: rgb(82, 173, 200)">link to paper</a>, <a href="https://github.com/flanagan-lab/flanagan-lab.github.io/blob/master/pdfs/Howell%20et%20al%202021%20-%20environmental%20DNA%20as%20a%20monitoring%20tool.pdf" target="_blank" rel="noreferrer noopener" aria-label="PDF (opens in a new tab)" style="color: rgb(82, 173, 200)">PDF</a>, <a href="https://sarahpflanagan.wordpress.com/2021/06/23/edna-antarctica/" target="_blank" rel="noreferrer noopener" aria-label="my summary (opens in a new tab)" style="color: rgb(82, 173, 200)">my summary</a>

 <strong>Flanagan SP</strong>, and Jones AG. 2019. The future of parentage analysis: Moving from microsatellites to SNPs and beyond. Molecular Ecology. 28(3): 544-567 doi: 10.1111/mec.14988 
    <a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/mec.14988" target="_blank" rel="noopener" style="color: rgb(82, 173, 200)">link to paper</a>, <a href="https://sarahpflanagan.wordpress.com/2019/01/21/a-review-of-parentage-analysis/" target="_blank" rel="noreferrer noopener" aria-label="my summary (opens in a new tab)" style="color: rgb(82, 173, 200)">my summary</a>
  
 Jost L, Archer F,<strong> Flanagan SP</strong>, Gaggiotti O, Hoban S, Latch E. 2018. Differentiation measures for conservation genetics. Evolutionary Applications. 11: 1139-1148. doi: 10.1111/eva.12590. 
    <a title="Jost_et_al-2018-Evolutionary_Applications" href="https://sarahpflanagan.files.wordpress.com/2018/02/jost_et_al-2018-evolutionary_applications.pdf" target="_blank" rel="noopener" style="color: rgb(82, 173, 200)">pdf</a>, <a href="http://onlinelibrary.wiley.com/doi/10.1111/eva.12590/full" target="_blank" rel="noopener" style="color: rgb(82, 173, 200)">link to paper</a>, <a href="https://sarahpflanagan.wordpress.com/2017/12/22/how-can-modern-genetic-tools-be-used-in-conservation-assessment-and-monitoring/" target="_blank" rel="noopener" style="color: rgb(82, 173, 200)">my summary</a>

  <strong><span class="author">Flanagan SP*</span></strong>, <span class="author">Forester BR*</span>, <span class="author">Latch EK</span>, <span class="author">Aitken SN</span>, <span class="author">Hoban S</span>. 2017. <span class="articleTitle">Guidelines for planning genomic assessment and monitoring of locally adaptive variation to inform species conservation</span>. <span class="journalTitle">Evolutionary Applications</span>. <span class="vol">11</span>: <span class="pageFirst">1035</span>–<span class="pageLast">1052</span>. doi: 10.1111/eva.12569. 
    <a title="Flanagan_et_al-2017-Evolutionary_Applications" href="https://sarahpflanagan.files.wordpress.com/2017/12/flanagan_et_al-2017-evolutionary_applications.pdf" target="_blank" rel="noopener" style="color: rgb(82, 173, 200)">pdf</a>, <a href="http://onlinelibrary.wiley.com/doi/10.1111/eva.12569/full" target="_blank" rel="noopener" style="color: rgb(82, 173, 200)">link to paper</a>, <a href="https://sarahpflanagan.wordpress.com/2017/12/22/how-can-modern-genetic-tools-be-used-in-conservation-assessment-and-monitoring/" target="_blank" rel="noopener" style="color: rgb(82, 173, 200)">my summary</a>

 
