---
title: "Do evolutionary mechanisms impact related populations and species in similar ways?"
collection: research
permalink: /research/popgen
show_date: false
excerpt: 'We use a variety of genetic tools to identify the primary evolutionary mechanisms impacting populations, including distinguishing between signatures of neutral and selective pressures. We are primarily interested in how spatially varying environments and selective pressures impact evolutionary processes. Most of the work to date has been done on the Gulf pipefish, but we are currently working on a broad range of organisms, including pipefish, penguins, and Phytophthora.'
---

Genomic data can be used to identify patterns of selection from the background patterns caused by other evolutionary processes (genetic drift, mutation, migration, recombination). Our lab uses these tools to ask questions about the relative importance of different types of selection, the impacts of spatial variation in environments, and to compare patterns of differentiation at the genomic level to differentiation at the phenotypic level. Our work on the Gulf pipefish is building a compelling that will help elucidate how different selection pressures impact the Gulf pipefish genome, and whether patterns of adaptation and sexual selection are repeatable across populations. 

That being said, the general dynamics of local adaptation, population differentiation, and novel trait evolution are best understood if multiple evolutionary trajectories are compared. To that end, the lab is developing comparative population genomics and transcriptomics studies using other Syngnathid fishes. Specifically, we have an ongoing project investigating the population genetics of <i>Stigmatopora nigra</i> across New Zealand. 

These same genetic tools can be used to understand the drivers of population decline. We are pursuing this question in a large but declining population of kororā, the little blue penguins (<i>Eudyptula minor</i>) in the Banks Peninsula. This collaborative project with [Pohatu Penguins](https://www.pohatu.co.nz/) and local rūnanga is using a different approach to obtaining genetic data, though: using environmental DNA from faeces and soil to identify prey items and genotype birds at specific genetic markers. We also plan to use the portable long-read sequencer, the Nanopore MinION, to generate whole-genome sequence data from blood samples collected while microchipping birds.

<img src='/images/DSC_2221.JPG'>

One really exciting and interesting aspect of the little blue penguin work is that the populations across New Zealand differ in their traits and their genetic clade (depending on your taxonomic preferences, which sub-species the populations belong to). In particular, in Banks Peninsula there is a specific morph called the white flippered penguin, which occurs only in a narrow range. We are working to understand how this trait evolved and the mechanisms allowing it to persist despite matings with the typical blue morph. 

Finally, the lab is also collaborating with Claudia Meisrimler to investigate the evolution of pathogenicity across strains of the pathogentic oomycete <i>Phytophtora cinnamomi</i>. In this work, we are using nanopore MinION sequencing data to generate genome assemblies for strains of this pathogen that were obtained from different plant hosts -- including kauri, pine trees, and avocados. We intend to identify the evolutionary mechanisms impacting the large set of genes referred to the 'effectome', which allow the oomycete to attack plant cells, with a long-term goal of applying this knowledge to reduce the harm this pathogen can cause. This research is led by MSc student Alex Cox.


Some relevant papers and theses from the lab include:

Howell LA, LaRue M, Flanagan SP (2021) Environmental DNA as a tool for monitoring Antarctic vertebrates. New Zealand Journal of Zoology. DOI: 10.1080/03014223.2021.1900299. <a href="https://doi.org/10.1080/03014223.2021.1900299" target="_blank" rel="noopener" style="color: rgb(82, 173, 200)">link to paper</a>, <a href="https://github.com/flanagan-lab/flanagan-lab.github.io/blob/master/pdfs/Howell%20et%20al%202021%20-%20environmental%20DNA%20as%20a%20monitoring%20tool.pdf" target="_blank" rel="noreferrer noopener" aria-label="PDF (opens in a new tab)" style="color: rgb(82, 173, 200)">PDF</a>, <a href="https://sarahpflanagan.wordpress.com/2021/06/23/edna-antarctica/" target="_blank" rel="noreferrer noopener" aria-label="my summary (opens in a new tab)" style="color: rgb(82, 173, 200)">my summary</a>

 Flanagan SP, Rose E, Jones AG (2021) The population genomics of repeated freshwater colonizations by Gulf pipefish. Molecular Ecology. DOI: 10.1111/mec.15841 <a href="http://doi.org/10.1111/mec.15841" target="_blank" rel="noopener" style="color: rgb(82, 173, 200)">link to paper</a>, <a href="https://github.com/flanagan-lab/flanagan-lab.github.io/blob/master/pdfs/Flanagan%20et%20al%202021%20-%20fwsw%20ACCEPTED.pdf" target="_blank" rel="noreferrer noopener" aria-label="PDF (accepted version)" style="color: rgb(82, 173, 200)">PDF</a>

Howell, L. 2021. Detecting Antarctic and New Zealand vertebrates using environmental DNA (eDNA). http://dx.doi.org/10.26021/10882. <a href="https://ir.canterbury.ac.nz/handle/10092/101828" target="_blank" rel="noopener noreferrer" style="color: rgb(82, 173, 200)">link</a>

Jost L, Archer F, Flanagan SP, Gaggiotti O, Hoban S, Latch E. 2018. Differentiation measures for conservation genetics. Evolutionary Applications . 00:1-10. https://doi.org/10.1111/eva.12590. <a title="Jost_et_al-2018-Evolutionary_Applications" href="https://sarahpflanagan.files.wordpress.com/2018/02/jost_et_al-2018-evolutionary_applications.pdf" target="_blank" rel="noopener">pdf</a>, <a href="http://onlinelibrary.wiley.com/doi/10.1111/eva.12590/full" target="_blank" rel="noopener">link to paper</a>, <a href="https://sarahpflanagan.wordpress.com/2017/12/22/how-can-modern-genetic-tools-be-used-in-conservation-assessment-and-monitoring/" target="_blank" rel="noopener">my summary</a> 

<span class="author">Flanagan SP</span>, <span class="author">Forester BR</span>, <span class="author">Latch EK</span>, <span class="author">Aitken SN</span>, <span class="author">Hoban S</span>. 2017. <span class="articleTitle">Guidelines for planning genomic assessment and monitoring of locally adaptive variation to inform species conservation</span>. <span class="journalTitle">Evolutionary Applications</span>. <span class="vol">00</span>: <span class="pageFirst">1</span>–<span class="pageLast">18</span>. https://doi.org/10.1111/eva.12569. <a title="Flanagan_et_al-2017-Evolutionary_Applications" href="https://sarahpflanagan.files.wordpress.com/2017/12/flanagan_et_al-2017-evolutionary_applications.pdf" target="_blank" rel="noopener">pdf</a>, <a href="http://onlinelibrary.wiley.com/doi/10.1111/eva.12569/full" target="_blank" rel="noopener">link to paper</a>, <a href="https://sarahpflanagan.wordpress.com/2017/12/22/how-can-modern-genetic-tools-be-used-in-conservation-assessment-and-monitoring/" target="_blank" rel="noopener">my summary</a> 

Flanagan SP, and Jones AG. 2017. Substantial differences in bias between single-digest and double-digest RAD-seq: A case study. Molecular Ecology Resources. <span class="vol">00</span>:<span class="pageFirst">1</span>–<span class="pageLast">17</span>. https://doi-org.proxy.lib.utk.edu:2050/10.1111/1755-0998.12734. <a href="http://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12734/full" target="_blank" rel="noopener">link to paper</a>, <a href="https://sarahpflanagan.wordpress.com/2017/12/13/rad-seq-in-pipefish-a-cautionary-tale/" target="_blank" rel="noopener">my summary</a>, <a href="https://github.com/spflanagan/SCA" target="_blank" rel="noopener">associated code</a>, <a href="https://github.com/spflanagan/gwscaR" target="_blank" rel="noopener">gwscaR</a>, <a href="https://doi.org/10.5061/dryad.qf916" target="_blank" rel="noopener">data</a>, <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRP096542" target="_blank" rel="noopener">genomic data</a>

Flanagan SP, and Jones AG. 2017. Constraints on the <em>F</em><sub>ST</sub>-heterozygosity outlier approach. Journal of Heredity. esx048. doi: 10.1093/jhered/esx048  <a title="Flanagan and Jones 2017 JHeredity" href="https://sarahpflanagan.files.wordpress.com/2018/01/flanagan-and-jones-2017-jheredity.pdf" target="_blank" rel="noopener">pdf</a>, <a href="https://academic.oup.com//jhered/article/doi/10.1093/jhered/esx048/3806805/Constraints-on-the-FSTHeterozygosity-Outlier?guestAccessKey=bb60bd3f-bdc2-4673-9b74-92efefd04db0" target="_blank" rel="noopener">link to paper</a>, <a href="https://sarahpflanagan.wordpress.com/2017/06/07/finding-limitations-with-common-analysis-methods-my-new-paper/" target="_blank" rel="noopener">my summary</a>, <a href="https://github.com/spflanagan/fsthet_analysis" target="_blank" rel="noopener">associated code</a>, <a href="http://datadryad.org/handle/10255/dryad.103085" target="_blank" rel="noopener">data</a> 

Flanagan SP, and Jones AG. 2017. Genome-wide selection components analysis in a fish with male pregnancy. Evolution.  71: 1096–1105. DOI: 10.1111/evo.13173 <a title="Flanagan and Jones 2017 Evolution" href="https://sarahpflanagan.files.wordpress.com/2018/01/flanagan-and-jones-2017-evolution.pdf" target="_blank" rel="noopener">pdf</a>, <a href="http://onlinelibrary.wiley.com/doi/10.1111/evo.13173/abstract;jsessionid=9B5AF79B09D88A653C1963FE810C7B31.f02t03" target="_blank" rel="noopener">link to paper</a>, <a href="https://sarahpflanagan.wordpress.com/2017/02/06/understanding-the-different-components-of-selection/" target="_blank" rel="noopener">my summary</a>, <a href="https://zenodo.org/record/200439" target="_blank" rel="noopener">associated code</a>, <a href="http://dx.doi.org/10.5061/dryad.5c1tj" target="_blank" rel="noopener">data</a>, <a href="https://trace.ncbi.nlm.nih.gov/Traces/sra_sub/sub.cgi?acc=SRP096542&focus=SRP096542&from=list&action=show:STUDY" target="_blank" rel="noopener">genomic data</a> 

Flanagan SP, Rose E, and Jones AG. 2016. Population genomics reveals multiple drivers of population differentiation in a sex-role-reversed pipefish. Molecular Ecology. 25(20): 5043-5072. doi: 10.1111/mec.13794 <a title="Flanagan et al 2016" href="https://sarahpflanagan.files.wordpress.com/2018/01/flanagan-et-al-2016.pdf" target="_blank" rel="noopener">pdf</a>, <a href="http://onlinelibrary.wiley.com/doi/10.1111/mec.13794/abstract" target="_blank" rel="noopener noreferrer">link to paper</a>, <a href="https://sarahpflanagan.wordpress.com/2016/11/04/population-genomics-what-is-it-and-why-should-you-care/" target="_blank" rel="noopener">my summary</a>, <a href="http://dx.doi.org/10.5061/dryad.38nh3" target="_blank" rel="noopener">data</a>, <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA327313" target="_blank" rel="noopener">genomic data</a>, <a href="https://zenodo.org/badge/latestdoi/51271165" target="_blank" rel="noopener">associated code</a> 

Flanagan SP, and Jones AG. 2015. Identifying signatures of sexual selection using genome-wide selection components analysis. Ecology and Evolution. 5(13): 2722-2744. doi: 10.1002/ece3.1546. <a href="https://sarahpflanagan.files.wordpress.com/2016/09/ece30005-2722.pdf">pdf</a>, <a href="https://sarahpflanagan.wordpress.com/2015/06/22/my-new-paper/">my summary</a>, <a href="http://dx.doi.org/10.5061/dryad.5k84d" target="_blank" rel="noopener">data</a>, <a href="https://github.com/spflanagan/gwsca_simulation_model" target="_blank" rel="noopener">associated code</a> 


